src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -177,25 +182,48 @@ private function isOtherServerProcessRunning($address)
     }
 
     /**
+     * Determine the absolute file path for the router script based on user input and the environment.
+     *
+     * @param InputInterface  $input  An InputInterface instance
+     * @param OutputInterface $output An OutputInterface instance
+     * @param string          $env    The application environment
+     *
+     * @return string|bool The absolute file path of the router script, or false on failure
+     */
+    private function determineRouterScript(InputInterface $input, OutputInterface $output, $env)
+    {
+        $router = $input->getOption('router');
```

Comment:
@xabbuh - I modified the function so that it takes the router argument directly in 898f97ab6bedbc544cb3eb8915978415f700f178.